test/integration/util/util.go

```diff
@@ -50,6 +52,14 @@ func (m *MinikubeRunner) RunCommand(command string, checkError bool) string {
 	return string(stdout)
 }
 
+func (m *MinikubeRunner) EnsureRunning() error {
+	if m.GetStatus() != "Running" {
+		m.RunCommand("start", true)
```

Comment:
RunCommand does not return an error as of right now, I believe it calls Fatalf instead.  Should I change it from using Fatalf to returning an error? (I think this change will affect other tests as well)